account_id_unit_test.go

```diff
@@ -35,3 +35,12 @@ func TestUnitAccountIDFromStringAlias(t *testing.T) {
 
 	assert.Equal(t, id.String(), id2.String())
 }
+
+func TestUnitChecksum(t *testing.T) {
+	ad1, err := _ChecksumParseAddress("1", "0.0.3")
+	require.NoError(t, err)
+	ad2, err := _ChecksumParseAddress("10", "0.0.3")
+	require.NoError(t, err)
+
+	require.NotEqual(t, ad1.correctChecksum, ad2.correctChecksum)
```

Comment:
Is this actually the case? I don't think we want 1 and 10 to equal the same checksum anymore, we actually want it to be different now.

Reply:
Ye this is the test for that.